daemon/algod/api/server/router_test.go

```diff
@@ -178,5 +178,6 @@ func (s *TestSuite) TestGetTransactionByID() {
 	assert.Equal(s.T(), callsBefore+1, s.calls)
 }
 func TestTestSuite(t *testing.T) {
+
```

Comment:
Do the tests in here need `partitiontest.PartitionTest(s.T())`?

Reply:
I think just this TestTestSuite function would need it, the other functions are all called from this test via the testify/suite feature